.github/workflows/update-prometheus.yml

```diff
@@ -0,0 +1,60 @@
+name: Update Prometheus
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "prometheus/**"
+
+jobs:
+  build-prometheus-image:
+    name: Build Docker image and push to ECR
+    environment: dev
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and tag image
+        uses: docker/build-push-action@v4
+        with:
+          context: ./prometheus
+          push: false
+          tags: ${{ github.ref_name }}:latest
```

Comment:
I don't think this is necessarily anything we need to worry about, but do we have a way to prove a heritage (e.g. this image came from X upstream version?). I think this applies to both this and the Grafana variant

Reply:
I think it's plausible that just the root docker-file version + commit hash is enough here, but figured it's worth a question at least!